.eslintrc.js

```diff
@@ -14,32 +14,33 @@ module.exports = {
   ],
   rules: {
     // Enable additional rules
-    'linebreak-style': [ 'error', 'unix' ],
+    'linebreak-style': ['error', 'unix'],
```

Comment:
There's a style somewhere that requires spaces ... perhaps that's js only.

Reply:
Nothing is enforcing in this file, so I'm just running autoformatting on it in my IDE.
Again, my preference is to `prettier` the heck outta everything, but happy to go with this minimal linting approach if it suits the majority of folks.

Reply:
I'm trying to bring some consistency to the `2` vs `'error'` thing at least. 🤷